src/server.rs

```diff
@@ -110,14 +111,15 @@ impl ServerPlugin {
 
     fn acks_receiving_system(
         mut acked_ticks: ResMut<AckedTicks>,
+        mut ticks_map: ResMut<TicksMap>,
         mut server: ResMut<RenetServer>,
         mut entity_map: ResMut<ClientEntityMap>,
     ) {
         for client_id in server.clients_id() {
             while let Some(message) = server.receive_message(client_id, REPLICATION_CHANNEL_ID) {
                 match bincode::deserialize::<RepliconTick>(&message) {
                     Ok(tick) => {
-                        let acked_tick = acked_ticks.clients.entry(client_id).or_default();
+                        let acked_tick = acked_ticks.0.entry(client_id).or_default();
```

Comment:
In places like this it requires `DerefMut` because it mutates the struct. But I didn't implement it to avoid letting users mutate these resources, only read (the resource is public).
Not sure if they are useful to users, BTW. So I could make it private and then I'm fine with adding `DerefMut`. What do you think?